docs/tracing.rst

```diff
@@ -249,24 +250,119 @@ runId              The workflow execution unique ID.
+                "multiqc_config": "/Users/sven1103/.nextflow/assets/nf-core/hlatyping/conf/multiqc_config.yaml",
+                "clusterOptions": false,
+                "cluster-options": false,
+                "enumerations": 1,
+                "beta": 0.009,
+                "prefix": "hla_run",
+                "base_index": "/Users/sven1103/.nextflow/assets/nf-core/hlatyping/data/indices/yara/hla_reference_",
+                "index": "/Users/sven1103/.nextflow/assets/nf-core/hlatyping/data/indices/yara/hla_reference_dna",
+                "custom_config_version": "master",
+                "custom_config_base": "https://raw.githubusercontent.com/nf-core/configs/master"
+            },
+            "workflow": {
+                "start": "2019-03-25T12:09:52Z",
+                "projectDir": "/Users/sven1103/.nextflow/assets/nf-core/hlatyping",
+                "manifest": {
+                    "nextflowVersion": ">=18.10.1",
+                    "defaultBranch": "master",
+                    "version": "1.1.4",
+                    "homePage": "https://github.com/nf-core/hlatyping",
+                    "gitmodules": null,
+                    "description": "Precision HLA typing from next-generation sequencing data.",
+                    "name": "nf-core/hlatyping",
+                    "mainScript": "main.nf",
+                    "author": null
+                },
+                "complete": null,
+                "profile": "docker,test",
+                "homeDir": "/Users/sven1103",
+                "workDir": "/Users/sven1103/git/nextflow/work",
+                "container": "nfcore/hlatyping:1.1.4",
+                "commitId": "4bcced898ee23600bd8c249ff085f8f88db90e7c",
+                "errorMessage": null,
+                "repository": "https://github.com/nf-core/hlatyping.git",
+                "containerEngine": "docker",
+                "scriptFile": "/Users/sven1103/.nextflow/assets/nf-core/hlatyping/main.nf",
+                "userName": "sven1103",
+                "launchDir": "/Users/sven1103/git/nextflow",
+                "runName": "shrivelled_cantor",
+                "configFiles": [
+                    "/Users/sven1103/.nextflow/assets/nf-core/hlatyping/nextflow.config"
+                ],
+                "sessionId": "7f344978-999c-480d-8439-741bc7520f6a",
+                "errorReport": null,
+                "scriptId": "2902f5aa7f297f2dccd6baebac7730a2",
+                "revision": "master",
+                "exitStatus": null,
+                "commandLine": "./launch.sh run nf-core/hlatyping -profile docker,test -with-weblog 'http://localhost:4567'",
+                "nextflow": {
+                    "version": {
+                        "minor": "03",
+                        "major": "19",
+                        "patch": "0-edge"
+                    },
+                    "build": 5119,
+                    "timestamp": "20-03-2019 19:29 UTC"
+                },
```

Comment:
The NF build timestamp does not look formatted as the other ts, doesn't it?

Reply:
no, it turns out to be a String, so I have to manually capture it and reformat it to ISO8601.

Reply:
ouch, there's no way to intercept a specific attribute while creating the json object?

Reply:
Yes, I can filter for attribute keys. Already pushed it and it works :)

Reply:
nice, tho I'm starting to think on the opportunity to keep timestamp as duration ie. as millis, what do you think about that?

Reply:
Hmm, I would rather expect the Nextflow compile timestamp to be of type `Date` than of type `Duration`.  Because it gives information about "when" and not "how long". Does this make sense?

Reply:
Maybe the best thing to do for now is to *sanitize* the `nextflow` map object just before the conversion to json, replacing it with another map in which `version` is a string, `build` remains as it is and `timestamp` is a date created using `new Date(Const.APP_TIMESTAMP)` 

WDUT ?

Reply:
sure, can make another static inner class `FlowBuild` which contains the sanitised representation.

Reply:
Would suggest to just use a map, because in a separate branch that I still need to merge the nextflow metadata is using it's own class. Then I will take to make it uniform with these changes.

Reply:
ok, sure